src/components/TalentPage/TalentPage.module.scss

```diff
@@ -0,0 +1,133 @@
+.proofs {
+    width: 100%;
+    height: 250px;
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+    justify-content: flex-start;
+    gap: 48px;
+    padding: 30px 10px 10px 40px;
+    color: #fff;
+    background: #111111;
+    border-radius: 16px;
+    //overflow: hidden;
+    transition: all 0.3s ease;
+}
+.info {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+}
+.container {
+    display: flex;
+    width: 100%;
+    height: 100%;
+    padding: 20px 20px;
```

Comment:
Either you need to remove the padding here, or add it at the bottom block.
![image](https://user-images.githubusercontent.com/101526474/230037360-698cb3b7-df30-40c5-ac60-5cc515c2a4ae.png)